input/examples/encounter-discharge-1.xml

```diff
@@ -38,6 +38,6 @@
     </location>
```

Comment:
Need to replace old location example with new.

Reply:
replaced with new location/murrabit-hospital